data_architect_misc/data_transformer/configs/AED_Ukraine/config.json

```diff
@@ -6,7 +6,7 @@
     "__comment__": "E.g., >> python transform.py -c configs/common_qa_config.json -i ./path/to/input/file/input_data_file.xlsx",
 
     "__comment__": "(Optional) Enter the sheet name of the input Excel file to read data from. Default is the first sheet.",
-    "input_sheet_name": "DateTemplate",
+    "input_sheet_name": "DataTemplate",
```

Comment:
If there is only one sheet in Ukraine datafiles, we can delete this "input_sheet_name" key in the config file and the transform module will read the sheet_index=0 by default. Maybe that's a more robust approach?

Reply:
Yes sire, you are right, I will delete this line of the config file